scene/3d/cpu_particles_3d.cpp

```diff
@@ -1465,6 +1465,7 @@ void CPUParticles3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_mesh"), &CPUParticles3D::get_mesh);
 
 	ClassDB::bind_method(D_METHOD("restart"), &CPUParticles3D::restart);
+	ClassDB::bind_method(D_METHOD("capture_aabb"), &CPUParticles3D::capture_aabb);
```

Comment:
For some reason CPU particles did not expose this method, while GPU did.
I needed it to generate AABB.

Reply:
But what about the 2D equivalent?

Reply:
Only GPU particles allow generating rect, so the code could handle a specific class. In 3D, since both types can generate AABB, I just used call().

Reply:
Okay then. If I could choose I'd rather have this exposed in another PR to keep it clean but it's also not that big of a deal.